src/main/java/org/apache/commons/imaging/formats/tiff/datareaders/DataReaderTiled.java

```diff
@@ -305,7 +307,15 @@ public TiffRasterData readRasterData(final Rectangle subImage)
             rasterWidth = width;
             rasterHeight = height;
         }
-        final float[] rasterData = new float[rasterWidth * rasterHeight];
+               float[] rasterDataFloat = null; // new float[rasterWidth * rasterHeight];
```

Comment:
Extra spaces? Can probably remove the spaces too, since the code is being used further down in the `if/else` block.

Reply:
done